tests/CMakeLists.txt

```diff
@@ -311,4 +310,13 @@ add_test_group(test_health_check
     ${UNIT_TESTS_DIR}/health_checks_test.cpp
     # src file(s)
     ${PROC_SRC_DIR}/application/health_checks/health_checks.c
-)
\ No newline at end of file
+)
+
+add_test_group(test_airdata_jacobian
```

Comment:
I feel like this should be rolled into the existing airdata tests, rather than making a new suite